deploy-review/solutions/microservice-schedule/src/main/java/com/redhat/training/conference/ScheduleResource.java

```diff
@@ -0,0 +1,128 @@
+package com.redhat.training.conference;
```

Comment:
Please normalize the style of all the applications you use in the same exercise scenario. So far I can see that this app uses one style, and the other one another.